mission-control-client/src/components/dashboard/user-dashboard/DashboardBanner.js

```diff
@@ -74,6 +73,5 @@ const mapStateToProps = state => {
 };
 
 export default connect(
-  mapStateToProps,
-  {}
+    mapStateToProps
```

Comment:
@zoverlvx we don't need the empty object when there are no actions to connect?

Reply:
Correct. It'll give us the dispatch by default.
https://react-redux.js.org/api/connect
`Conventionally called mapDispatchToProps, this second parameter to connect() may either be an object, a function, or not supplied.

Your component will receive dispatch by default, i.e., when you do not supply a second parameter to connect()`